README.md

```diff
@@ -2,6 +2,12 @@
 
 ## Quick Start
 
+### Option A: Using Docker (Recommended for macOS and Windows)
+
+For using Docker to build and run StarryOS, see [`docker/README.md`](docker/README.md).
+
+### Option B: Native Installation
```

Comment:
Split a `Quick Start (Docker)` section and a `Quick Start (Native)` section. Or you may want to change the titles for the subsections in `Option B: Native Installation` to h4.